samples/qa-a11y/src/views/Keypad.js

```diff
@@ -0,0 +1,22 @@
+import Keypad from '@enact/agate/Keypad';
+import React from 'react';
+
+import Section from '../components/Section';
+
+import appCss from '../App/App.module.less';
+
+const KeypadView = () => (
+	<>
+		<Section title="Default">
+			<Keypad alt="Normal" />
+			<Keypad alt="Disabled" disabled />
+		</Section>
+
+		<Section className={appCss.marginTop} title="Aria-labelled">
+			<Keypad alt="Aria-labelled" aria-label="This is a Label." />
```

Comment:
I think we should think about how we provide aria-label for this component fist.
:duh:

Reply:
I think it's out of this ticket scope but I saw
```Uncaught TypeError: this.props.onChange is not a function
    at handleInputValue (Keypad.js:304)
    at onKeyButtonClick (Keypad.js:181)
    at forward (handle.js:391)
    at _curryN.js:37
    at _arity.js:6
    at adaptEvent (handle.js:696)
    at handle (handle.js:122)
    at Object.prepareHandleArgs [as onClick] (handle.js:213)
```

Reply:
Removed the aria-labelled section first.

Reply:
I wrote it in the PR description.